### PR TITLE
update docs to reflect native html properties can be used with input

### PR DESCRIPTION
**Pull request by jacoblogan**
#### Description of changes

Update Documentation pages to reflect that native HTML tag properties can be used with Amplify UI components.

#### Checklist

- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zchenwei — APPROVED** [review]
LGTM :shipit:

**Review by joebuono — COMMENTED** [review]
I'm wondering if each input primitive deserves its own little section. For example:

### Native HTML properties

The `Button` primitive will accept any of the native HTML properties that a `<button>` element accepts. 

Then include an example showing how these native props can be passed. 

Thoughts?

File: docs/src/pages/components/button/react.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { IconButtonExample, ButtonThemeExample } from './examples';
 
 ## Usage
 
-Import the Button primitive and styles.
+Import the `Button` component. The `Button` component will also accept any of the native HTML properties that are accepted by a `button` tag.
```
Comment (reesscot) [inline docs/src/pages/components/button/react.mdx:13]: Should we provide a link to MDN here?

**Review by reesscot — APPROVED** [review]

**Comment by reesscot** [comment]
> I'm wondering if each input primitive deserves its own little section. For example:
> ### Native HTML properties
> 
> The `Button` primitive will accept any of the native HTML properties that a `<button>` element accepts.
> 
> Then include an example showing how these native props can be passed.
> 
> Thoughts?

I like this idea a lot. We could make it as a reusable component and pass the component name and tag names into the component. It could also include a link to the MDN docs.

**Comment by jacoblogan** [comment]
Closing this pull request as it has been replaced with https://github.com/aws-amplify/amplify-ui/pull/1516

**Pull request closed without merging**